lib/Parser/CharTrie.cpp

```diff
@@ -6,10 +6,12 @@
 
 namespace UnifiedRegex
 {
+    const int CharTrie::initCapacity;
```

Comment:
Header file has 'static const int initCapacity = 4`. It works by init in header file and not here?